apps/contacts/interface.html

```diff
@@ -7,6 +7,10 @@
     <link rel="stylesheet" href="../../css/spectre-icons.min.css">
     <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A==" crossorigin="anonymous">
     <link rel="stylesheet" href="https://unpkg.com/leaflet-control-geocoder/dist/Control.Geocoder.css">
+    <script type="module">
+      import vcf from 'https://cdn.jsdelivr.net/npm/vcf@2.1.1/+esm'
```

Comment:
I had a look, doesn't seem that unpkg supports bundling / there's no dist of `vcf`, so nice find here